lib/Weasel/Driver/Selenium2.pm

```diff
@@ -164,6 +164,17 @@ sub stop {
     $self->started(0);
 }
 
+=item find_all_displayed
+
+=cut
+
+sub find_all_displayed {
+    my $self = shift @_;
+    my @rv = $self->find_all(@_);
+    @rv = grep { $_->is_displayed } @rv;
```

Comment:
You yourself said that "is_displayed" is defined as "being part of the DOM". Then, what value does this function add?

Reply:
@ylavoie ping